### PR TITLE
added modal component

### PR DESCRIPTION
**Pull request by pariyar07**

### CODE REVIEW

**Review by psychoactiv — APPROVED** [review]

**Comment by psychoactiv** [comment]
It's good to go

File: styles/modal.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+.btn-modal {
+    font-size: 1.5rem;
+    padding: 0.5rem 4rem;
+    border: 1px solid white;
+    background: black;
+    color: white;
+    border-radius: 10px;
+}
+
+.btn-modal:hover {
```
Comment (yogesh-) [inline styles/modal.css:73]: use color variables.

File: styles/modal.css [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    padding: 0.5rem 1.2rem;
+    color: white;
+    border-radius: 5px;
+}
+
+.modal-btns.cancel:hover {
+    background-color: rgb(150, 150, 150);
+    color: white;
+    border: 1px solid rgb(150, 150, 150);
+}
```
Comment (yogesh-) [inline styles/modal.css:110]: use color varaibles

**Review by yogesh- — COMMENTED** [review]
LGTM

**Pull request merged**